src/AppContainer.jsx

```diff
@@ -82,13 +84,16 @@ const AppContainer = ({ store, lang, history, client }) => {
                     <BanksProvider client={client}>
                       <SelectionProvider>
                         <StoreURLProvider>
-                          <MuiCozyTheme>
-                            <RealTimeQueries doctype={TRIGGER_DOCTYPE} />
-                            <RealTimeQueries doctype={ACCOUNT_DOCTYPE} />
-                            <RealTimeQueries doctype={TRANSACTION_DOCTYPE} />
-                            <RealTimeQueries doctype={GROUP_DOCTYPE} />
-                            <Router history={history} routes={AppRoute()} />
-                          </MuiCozyTheme>
+                          <CozyConfirmDialogProvider>
+                            <MuiCozyTheme>
+                              <RealTimeQueries doctype={TRIGGER_DOCTYPE} />
+                              <RealTimeQueries doctype={ACCOUNT_DOCTYPE} />
+                              <RealTimeQueries doctype={COZY_ACCOUNT_DOCTYPE} />
+                              <RealTimeQueries doctype={TRANSACTION_DOCTYPE} />
+                              <RealTimeQueries doctype={GROUP_DOCTYPE} />
+                              <Router history={history} routes={AppRoute()} />
+                            </MuiCozyTheme>
+                          </CozyConfirmDialogProvider>
```

Comment:
Should `MuiCozyTheme` be above `CozyConfirmDialogProvider`? Otherwise dialogs will not have access to the theme as far as i understand.

Reply:
Totally right! Can you create the fix?